### ISSUE TITLE
Scope of the vocabulary 

### ISSUE THREAD

**Issue by paul2keller** [lrosenthol/opt-out-vocab#6]
> On 15 Jul 2024, at 16:07, Giulia Marangoni <giulia.marangoni@aie.it> wrote:
> 
> However, then (par.3.2) it states: "The TDM category is the overarching category that includes all of the others. This is because all of those other activities are considered to be forms of TDM. As such, if a rightsholder opts out of TDM, they are opting out of all of the other categories as well.” This last sentence should be deleted in my view, as it moves from a purely technical description of TDM activities to what is covered by opt-out solutions, which may differ in scope. For instance, search and discovery is not in the scope of the broad reservation expressed by TDM Rep.

So I think this is a fundamental misunderstanding of what we are trying to do with this vocabulary. The vocabulary is intended to ensure that opt-outs have the same effect across all opt-out solutions. Given this the vocabulary needs to be unambiguous in describing the relationships between the different concepts. 

The vocabulary should describe these relationships and as such describes the scope of these terms. If each opt out solution can describe its own scope separate from the vocabulary than the whole vocabulary cannot serve its intended purpose.

If we agree that "The TDM category is the overarching category that includes all of the others” which I think we did in Brussels then it logically follows that there are two ways to express the fact that TDM-REP currently considers “search and discovery” out of scope.

1. We defined the TDM category to not include search and discovery
2. You map the current TDM-REP reservation to a combination of  `TDM = reserved` + `Search and Discover = allowed`

**Comment by giuliamarangoni** [comment]
The main issue on the scope of the vocabulary that I see is that current draft of the vocabulary defines the broad "TDM category" according to the definition by art.4 of the DSM directive, then it includes search (and discovery) under it. This suggests an interpretation of the DSM directive on which there is no consensus among stakeholders, as repeatedly discussed in the Brussels workshop. 

I proposed via mail a couple of options, which I report here, to find a common ground for developing the vocabulary in way to reconcile different positions: 

a) to create a higher-level TDM category which is broader then TDM activities defined under art.4; such a broader category may include both TDM-art.4 usages and usages for search (and discovery). Search and discovery can fall within such higher level TDM category, but not under TDM usages falling under art-4 definition.

b) Alternatively, If there is a preference to keep the scope of the vocabulary as it is in the current draft, where the "TDM category" equals to the TDM definition by art.4 of the DSM, I ask you to exclude “search and discovery” from the hierarchy (which means to put search and discovery outside the big TDM box in the figure, in a separate box), because, as said before, there is no consensus on such interpretation of the DSM Directive.

**Comment by paul2keller** [comment]
Thank you @giuliamarangoni for your comments. Here is how I would suggest we address this issue. This is a bit long, but I think it is important that we get this right. TL;DR I think we should go with your option (b).

As we said in the last policy brief, we think it is essential that the vocabulary includes an option that allows rightholders to reserve rights to the extent that they can be reserved under Article 4(3). Not to do so would mean that the vocabulary would not work for tools that allow rightholders to make a full Article 4(3) reservation. Second, I do not think it makes sense to offer a category that is broader, since under EU copyright law such rights would either be covered by the standard copyright (and thus would not need to be reserved) or they would not be covered by copyright (and thus could not be reserved because there is no underlying right).

We (and thus the vocabulary) have to accept that the exact scope of what is covered by TDM under the CDSMD is unclear. Ultimately this is for the CJEU to decide, and in the absence of that certainty, the best thing we can do is to stick to the exact wording of the TDM definition from the CDSM Directive. If we do that, the vocabulary will remain aligned with a potentially evolving legal definition of the concept of TDM.

All of this means that your option (a) for the vocabulary is not viable.

Regarding your option (b), I do not think we can say with certainty at this point whether or not search and discovery is covered by the TDM definition. I know that you have had conversations with the Commission in which they have said that they do not believe that it is, and it certainly was not the intent of anyone involved in drafting the rules, but ultimately it is for the courts to decide. It is relatively clear to me that some search and discovery services use TDM, but I think it will be really hard to draw an exact line here. It is also clear to me that no one wants to have a definition that includes the simple indexing of web pages for search purposes that has underpinned Internet search for the last 2.5 decades. 

In my summary remarks at the workshop, I concluded that we would develop a vocabulary that includes the overarching category (TDM) and a number of specific use cases where there is less clarity on how to define them. Based on the discussion here, I think we can conclude that we do not currently have a sufficiently stable understanding of the category "search and discovery" to include such a category in the vocabulary. 

What I would propose is that we do not include this use case in the vocabulary for the time being, which would mean that in addition to the overarching "TDM" category, we currently have "Generative AI Training" and "AI Training". In addition, I would suggest that we either include a short note in the abstract that we consider simple indexing of online resources for search purposes to be outside the scope of the vocabulary. 

I hope this approach addresses your concerns and works for everyone else. Please let us know what you think @lrosenthol @sposth

**Comment by lrosenthol** [comment]
I think that Search & Discovery needs to be defined category, as it clearly aligns with how `Robots.txt` works.  But I do recognize this debate on whether it falls into TDM or not, so I will restructure to move it into its own category - thus flattening out the scenarios a bit...

**Comment by paul2keller** [comment]
I do not think that aligning with robots.txt is our primary concern at this stage. But on substance I do not think that this is correct: robots.txt is usage agnostic. in its current form it either allows crawling or it does not allow crawling but it does not care about the purpose of the crawl. From my perspective the purpose of our vocabulary is that certain usage forms can then be mapped to robots.txt with the main candidate for such a mapping benig `generative-ai-taining` (which seems roughly equivalent with what the -extended bots (and specialized bots like GPTBot) do. So I do not think that we really need `search and discovery`.

**Comment by lrosenthol** [comment]
> I do not think that aligning with robots.txt is our primary concern at this stage. 

I agree, but we can't shouldn't ignore it as a target use case/scenario.

>But on substance I do not think that this is correct: robots.txt is usage agnostic. 

Technically yes, but functionally no.
